Qt/InstaDam/Selector/selectItem.h

```diff
@@ -80,6 +80,7 @@ class SelectItem : public QGraphicsItem {
     virtual void resizeItem(const int vertex, QPointF &shift) = 0;
     virtual void rotate(const QPointF &from, const QPointF &to) = 0;
     virtual void updatePen(QPen pen) = 0;
+    void setOpacity(qreal val) {QGraphicsItem::setOpacity(val);}
```

Comment:
5: Cannot find the User Story that the change implements.

Reply:
Please see the revised user story list here (User Story 37) https://wiki.illinois.edu/wiki/display/CS428SP19/IDM+-+Iteration+2